configs/example/riscv/fs_linux.py

```diff
@@ -135,20 +140,8 @@ def generateDtb(system):
 
 # ----------------------------- Add Options ---------------------------- #
 parser = argparse.ArgumentParser()
-Options.addCommonOptions(parser)
+Options.addCommonOptions(parser, ISA.RISCV)
```

Comment:
No need to make any changes, but since this script uses `common/Options.py` it should probably be moved to "deprecated." 

The maintainers are explicitly not maintaining anything that uses `common/Options.py`. That said, I appreciate the community continuing to make these scripts work.